pkg/deploy/elbv2/load_balancer_manager.go

```diff
@@ -75,6 +76,12 @@ func (m *defaultLoadBalancerManager) Create(ctx context.Context, resLB *elbv2mod
 		return elbv2model.LoadBalancerStatus{}, err
 	}
 
+	if resLB.Spec.Type == elbv2model.LoadBalancerTypeNetwork && resLB.Spec.SecurityGroupsInboundRulesOnPrivateLink != nil {
```

Comment:
is this `EnforceSecurityGroupInboundRulesOnPrivateLinkTraffic` can only be set with this SetSecurityGroups call after LB creation?

Reply:
Yes, we can only set it after LB creation